src/WindowsTracing/TracerImpl.cpp

```diff
@@ -23,4 +31,29 @@ void TracerImpl::Stop() {
   krabs_tracer_ = nullptr;
 }
 
+void TracerImpl::SendModulesSnapshot() {
+  uint32_t pid = capture_options_.pid();
+  ModulesSnapshot modules_snapshot;
+  modules_snapshot.set_pid(pid);
+  modules_snapshot.set_timestamp_ns(orbit_base::CaptureTimestampNs());
+
+  std::vector<Module> modules = orbit_windows_utils::ListModules(pid);
+  if (modules.empty()) {
+    ERROR("Unable to load modules for %u", pid);
+    return;
+  }
+
+  for (const Module& module : modules) {
+    ModuleInfo* module_info = modules_snapshot.add_modules();
+    module_info->set_name(module.name);
+    module_info->set_file_path(module.full_path);
+    module_info->set_address_start(module.address_start);
+    module_info->set_address_end(module.address_end);
+    module_info->set_build_id(module.build_id);
+  }
+
+  // Will uncomment when https://github.com/google/orbit/pull/2936 is merged.
+  // listener_->OnModulesSnapshot(std::move(modules_snapshot));
```

Comment:
I had looked into it quickly but only had the branches for the main repo available, I didn't go further. Next time I'll try to enter my fork. Good to know, thanks.

Reply:
Yeah, seems that you can not change the target repo once the PR is created (you can at creation time). On the next "chain" of PRs, it might be worth to actually create feature branches on google/orbit instead of the fork to allow such a workflow (you can still re-target a PR to a different branch).